person.rb

```diff
@@ -0,0 +1,27 @@
+class Person
+  def initialize(age, name = 'Unknown', _parent_permission: true)
+    @id = Random.rand(1..100)
+    @name = name
+    @age = age
+  end
+
+  attr_reader :id, :name, :age
```

Comment:
- [ ] Following the project requirements, you should add setters for @name and @age.`:name` and `:age` instance variables are the only ones that should be able to be modified that's why they should be defined as attr_accessors (Getters and Setters).